diff -ruN a/drivers/media/usb/uvc/uvc_driver.c b/drivers/media/usb/uvc/uvc_driver.c
--- a/drivers/media/usb/uvc/uvc_driver.c
+++ b/drivers/media/usb/uvc/uvc_driver.c
@@ -2461,6 +2461,38 @@
 	  .bInterfaceSubClass	= 1,
 	  .bInterfaceProtocol	= UVC_PC_PROTOCOL_15,
 	  .driver_info		= (kernel_ulong_t)&uvc_ctrl_power_line_limited },
+
+	/**
+	   * Fix for the problem with cameras on Acer Nitro 5 Series.
+	   * Thanks to @Giuliano69 for providing the fix
+	  */
+	/* Quanta ACER HD User Facing  0x4035 - Experimental */
+	{ .match_flags = USB_DEVICE_ID_MATCH_DEVICE |
+			 USB_DEVICE_ID_MATCH_INT_INFO,
+	  .idVendor = 0x0408,
+	  .idProduct = 0x4035,
+	  .bInterfaceClass = USB_CLASS_VIDEO,
+	  .bInterfaceSubClass = 1,
+	  .bInterfaceProtocol = UVC_PC_PROTOCOL_15,
+	  .driver_info = (kernel_ulong_t) &
+			 (const struct uvc_device_info){
+				 .uvc_version = 0x010a,
+			 } },
+
+	/* Quanta ACER HD User Facing 4033 - Experimental !! */
+	{ .match_flags = USB_DEVICE_ID_MATCH_DEVICE |
+			 USB_DEVICE_ID_MATCH_INT_INFO,
+	  .idVendor = 0x0408,
+	  .idProduct = 0x4033,
+	  .bInterfaceClass = USB_CLASS_VIDEO,
+	  .bInterfaceSubClass = 1,
+	  .bInterfaceProtocol = UVC_PC_PROTOCOL_15,
+	  .driver_info = (kernel_ulong_t) &
+			 (const struct uvc_device_info){
+				 .uvc_version = 0x010a,
+			 } },
+
+
 	/* LogiLink Wireless Webcam */
 	{ .match_flags		= USB_DEVICE_ID_MATCH_DEVICE
 				| USB_DEVICE_ID_MATCH_INT_INFO,
